app/src/main/java/org/mozilla/focus/biometrics/BiometricAuthenticationHandler.kt

```diff
@@ -0,0 +1,181 @@
+package org.mozilla.focus.biometrics
+
+import android.arch.lifecycle.Lifecycle
+import android.arch.lifecycle.LifecycleObserver
+import android.arch.lifecycle.OnLifecycleEvent
+import android.arch.lifecycle.ProcessLifecycleOwner
+import android.content.Context
+import android.os.Build
+import android.security.keystore.KeyGenParameterSpec
+import android.security.keystore.KeyPermanentlyInvalidatedException
+import android.security.keystore.KeyProperties
+import android.support.annotation.RequiresApi
+import android.support.v4.hardware.fingerprint.FingerprintManagerCompat
+import android.support.v4.os.CancellationSignal
+import java.io.IOException
+import java.security.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.

Reply:
This happens when I individually import all the different exceptions because I'm actually using all the java.security.* imports, I believe.